src/hotspot/os_cpu/linux_riscv/riscv_hwprobe.cpp

```diff
@@ -163,7 +165,18 @@ void RiscvHwprobe::add_features_from_query_result() {
     VM_Version::ext_C.enable_feature();
   }
   if (is_set(RISCV_HWPROBE_KEY_IMA_EXT_0, RISCV_HWPROBE_IMA_V)) {
-    VM_Version::ext_V.enable_feature();
+    // Linux signal return bug when using vector with vlen > 128b in pre 6.9.
+    long major, minor;
+    os::Linux::kernel_version(&major, &minor);
+    if (!(major > 6 || (major == 6 && minor >= 9))) {
+      LogMessage(os) log;
+      if (log.is_info()) {
```

Comment:
`log_info` could make the code simpler.

Reply:
This is a multi-line log therefore I need the LogMessage.
Without it it's not certain they are logged together or with the same decorators.
The log level could even be changed between two log_info so only one of the lines are logged.